analyses/molecular-subtyping-CRANIO/00-craniopharyngiomas-molecular-subtype.Rmd

```diff
@@ -62,11 +63,11 @@ output_subtype <- file.path(results_dir, "CRANIO_molecular_subtype.tsv")
 # Inputs
 ## Read in metadata
 metadata <-
-  read_tsv(file.path(root_dir, "data", "pbta-histologies-base.tsv")) %>%
+  read_tsv(file.path(root_dir, "data", "pbta-histologies-base-adapt.tsv")) %>%
   filter(pathology_diagnosis == "Craniopharyngioma")
```

Comment:
```suggestion
  read_tsv(file.path(root_dir, "data", "pbta-histologies-base-adapt.tsv")) %>%
  filter(kf_visibility == TRUE & pathology_diagnosis == "Craniopharyngioma")
```

Reply:
@komalsrathi I am noticing a lot of new samples coming up here and some are not visible - can you add this filter to all modules and rerun? thanks

Reply:
@jharenza do you know if the PBTA base file that will be made available in the future versions under `data/` will not have `kf_visibility FALSE` samples just like how they have in OpenPBTA's 'data/pbta-histologies-base.tsv'? If so it would make sense for me to make only one change i.e. keep `kf_visibility == TRUE` samples in `/data/pbta-histologies-base-adapt.tsv`.

Reply:
that was a filtering @kgaonkar6 or someone who pulled the file must have been doing, so I think all samples will always be there, which brings me to another point. this file will likely be renamed `histologies-base-adapt.tsv` in the future since it will soon contain all of the other cohorts as well, so for now, we might also change that filename.

Reply:
Ok I can change the file name and the most optimal thing for the long run would be to limit the file to only samples with kf_visibility to TRUE unless the FALSE samples are going to be used for or benefit any analyses. 

So unless there is an objection, I'll rename and filter the file to TRUE samples, store under data/ and reference as such in these modules. I'll rerun and let you know.

Reply:
Sure, we will just need to create this extra step somewhere upstream.